### PR TITLE
Stop the game after having winners

### LINKED ISSUE #2401: Stop the game after having winners
### Environment
* **FreeOrion Version**: 0.4.8, master
* **Fetched as** 
  * Binary release
  * Weekly development build
  * Compiled from source

### Description
Currently the game continues after some players wins the game.

### Expected Result
Server notifies all player about winners and their victory reason.
Server shutdowns or went back to lobby depending on mode.

### Steps to reproduce
* Win the game.
* Make next turns.

### PR DESCRIPTION
**Pull request by o01eg**
Fixes #2401

### CODE REVIEW

**Comment by geoffthemedio** [comment]
Should note in the option description for the server timeout that it will stop cycling turns automatically once a player has won.

**Comment by o01eg** [comment]
@geoffthemedio I've added note about it.

File: default/stringtables/en.txt [hunk 1]
```diff
@@ -1436,7 +1436,7 @@ OPTIONS_DB_FIRST_TURN_TIME
 Absolute time point in format "2019-01-20 11:59:59" in UTC. If empty, first turn advance will happen after expiring interval. Requires fixed interval to be enabled.
 
 OPTIONS_DB_TIMEOUT_INTERVAL
-Maximum interval in seconds between turn advances. If 0, turn advances by timeout are disabled, and turns will only advance once all players have ended their turn.
+Maximum interval in seconds between turn advances. If 0, turn advances by timeout are disabled, and turns will only advance once all players have ended their turn and no one win the game.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: Does it not also stop the turn advance by timer if someone has won?

Comment (o01eg) [reply default/stringtables/en.txt]: Fixed.

File: default/stringtables/en.txt [hunk 2]
```diff
@@ -1436,7 +1436,7 @@ OPTIONS_DB_FIRST_TURN_TIME
 Absolute time point in format "2019-01-20 11:59:59" in UTC. If empty, first turn advance will happen after expiring interval. Requires fixed interval to be enabled.
 
 OPTIONS_DB_TIMEOUT_INTERVAL
-Maximum interval in seconds between turn advances. If 0, turn advances by timeout are disabled, and turns will only advance once all players have ended their turn.
+Maximum interval in seconds between turn advances if no one player win the game. If 0, turn advances by timeout are disabled, and turns will only advance once all players have ended their turn.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: Maybe `Maximum interval in seconds between turn automatic advances. If 0, turn advances by timeout are disabled, and turns will only advance once all players have ended their turn. Automatic turn advance is also disabled after any player has won the game.`

Comment (o01eg) [reply default/stringtables/en.txt]: I've used it.

**Pull request merged**